src/bot/commands.rs

```diff
@@ -275,6 +275,23 @@ pub trait Command {
         telegram_client::api().clone()
     }
 
+    fn list_subscriptions(db_connection: &mut PgConnection, message: &Message) -> String {
```

Comment:
you should modify the list_subscriptions command. this function is not needed

Reply:
okay